### PR TITLE
PUBDEV-5454 deeplearningparametersv3 cannot read in integer array

### PR DESCRIPTION
**Pull request by wendycwong**
Strip away excess quotes around numeric arrays in order to be able to correctly instantiate them back.

### CODE REVIEW

File: h2o-core/src/main/java/water/api/Schema.java [hunk 1]
```diff
@@ -441,6 +448,8 @@ public S fillFromParms(Properties parms, boolean checkRequiredFields) {
         splits = new String[] { s.trim() };
       }
 
+      if (afclz==int.class || afclz==double.class || afclz==float.class)
```
Comment (michalkurka) [inline h2o-core/src/main/java/water/api/Schema.java]: Was this a bug? Can there be a test to demonstrate the fix?

Comment (wendycwong) [reply h2o-core/src/main/java/water/api/Schema.java]: Yes, it is a bug.  Without my fix, the newly added DL example will fail to run.

Comment (michalkurka) [reply h2o-core/src/main/java/water/api/Schema.java]: Can you please investigate how does the JSON request look like? Could you please paste it here for reference?

**Comment by wendycwong** [comment]
Michal K will take care of this one.

**Pull request closed without merging**